### ISSUE TITLE
Add a responsive Login Page 

### ISSUE THREAD

**Issue by shashank-iter** [Web3-Bharat-ITER/UI-Component-Library#11]
### The main task is to add a simple responsive login page, that should contain at least two fields for example username and password and a login button. You can show you creativity to add some extra design to it.

**Comment by lakshyaagr1105** [comment]
Hey @shashank-iter, I am Lakshya and i would like to work on this task. So, please assign it to me.

**Comment by shashank-iter** [comment]
@lakshyaagr1105  Okay sure assigning it you. Kindly read the contribution guidelines and add a screenshot of you work while making a PR.

**Comment by lakshyaagr1105** [comment]
@shashank-iter kindly merge the pull request

**Comment by MangEs23** [comment]
hi @shashank-iter can i still work on this?

**Comment by shashank-iter** [comment]
Yeah sure you can do that. Make you code it properly and try to show some creativity. Kindly add a screenshot while making a PR.

**Comment by hafidzzakky** [comment]
Hey @shashank-iter , I am interested in making a Responsive Login Page
Can you assign me ? i think i can add this feature to neomorphic login page?

**Comment by shashank-iter** [comment]
@hafidzzakky yes you can surely work on this issue. Don't forget to attach the screenshot while making the PR.

**Comment by SharmilaChoudhary** [comment]
hey @hafidzzakky  
I would like to work on this issue please assign it to me.

**Comment by shashank-iter** [comment]
@SharmilaChoudhary , @Jyo561Python and @shashank-iter are maintaining this repository.

**Comment by Aniketmishra0** [comment]
Hello Sir,
Can You assign this issue to me?

**Comment by Jyo561Python** [comment]
@Aniketmishra0 please get completed with one issue then only we can assign you another one.
